src/Catalyst.Core.Modules.Consensus.Tests/UnitTests/Deltas/DeltaTransactionRetrieverTests.cs

```diff
@@ -26,40 +26,42 @@
 using System.Linq;
 using Catalyst.Abstractions.Mempool;
 using Catalyst.Core.Lib.DAO;
-using Catalyst.Core.Lib.Extensions.Protocol.Wire;
+using Catalyst.Core.Lib.DAO.Transaction;
+using Catalyst.Core.Lib.Extensions;
 using Catalyst.Core.Modules.Consensus.Deltas;
-using Catalyst.Protocol.Wire;
+using Catalyst.Core.Modules.Hashing;
+using Catalyst.Protocol.Transaction;
 using Catalyst.TestUtils;
 using FluentAssertions;
 using NSubstitute;
+using TheDotNetLeague.MultiFormats.MultiHash;
 using Xunit;
 
 namespace Catalyst.Core.Modules.Consensus.Tests.UnitTests.Deltas
 {
     public sealed class DeltaTransactionRetrieverTests
     {
-        private readonly IList<TransactionBroadcast> _transactions;
+        private readonly IList<PublicEntry> _transactions;
         private readonly DeltaTransactionRetriever _transactionRetriever;
 
         public DeltaTransactionRetrieverTests()
         {
+            var hashProvider = new HashProvider(HashingAlgorithm.GetAlgorithmMetadata("blake2b-256"));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'hashProvider'.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4562326)